examples/nodejs_ibmcloud_operator/README.md

```diff
@@ -118,8 +81,28 @@ there the required backed service is not configured.
 Now we use the IBM Cloud Operator that the cluster admin has installed. To create a language translator instance just create a [`Service` custom resource for languagee translator](./language-translator.yaml) in the `service-binding-demo` namespace called `mytranslator` along with [a `Binding` custom resource](./language-translator-binding.yaml) by running:
 
 ``` shell
-oc apply -f language-translator.yaml
-oc apply -f language-translator-binding.yaml
+oc apply -f - << EOD
+---
+apiVersion: ibmcloud.ibm.com/v1alpha1
+kind: Service
+metadata:
+  name: mytranslator
+  namespace: service-binding-demo
+spec:
+  serviceClass: language-translator
+  plan: lite
+---
+apiVersion: ibmcloud.ibm.com/v1alpha1
+kind: Binding
+metadata:
+  name: mytranslator-binding
+  namespace: service-binding-demo
+  annotations:
+    service.binding/url: path={.status.secretName},objectType=Secret,sourceValue=url
```

Comment:
sourceValue seems to be redundant. I think it would be taken as URL as it is mentioned in service.binding/url unless we want  a different reference name

Reply:
We only want to bind just the `url` and `apikey` here from the service, nothing else... When you do not specify the `sourceValue` the entire secret will be bound...